.gitignore

```diff
@@ -14,6 +14,7 @@ test_logins.txt
 */templates_c/*.tpl.php
 */admin/templates_c/*.tpl.php
 */admin/templates_c/
+LEAF_Request_Portal/templates_history/*
```

Comment:
Were there some files in here that were required for Juan's template history to work?

Reply:
If it's for testing purposes I would recommend using a global gitignore file so that only what is absolutely needed for the ticket to work is in the repo gitignore. It's what I'm doing for testing with multiple portals.